frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -0,0 +1,10 @@
+import { UseK8sWatchResource, UseK8sWatchResources } from './api-types';
+
+export * from './api-types';
+
+const MockImpl = () => {
+  throw new Error('You need to configure webpack externals to use this function at runtime.');
+};
+
+export const useK8sWatchResource: UseK8sWatchResource = MockImpl;
+export const useK8sWatchResources: UseK8sWatchResources = MockImpl;
```

Comment:
I'd like to see us bring the actual code into the SDK package instead of doing this work around.